### PR TITLE
fix(frontend): add fiche url when submiting feedback

### PR DESCRIPTION
**Pull request by lionelB**
ajout de l'url de la page consultée fors d'une remontée utilisateur via le formulaire de feedback
fixe #316

### CODE REVIEW

File: packages/code-du-travail-frontend/src/common/Answer.js [hunk 1]
```diff
@@ -72,6 +72,7 @@ class Answer extends React.Component {
               isOpen={this.state.modalVisible}
               closeModal={this.closeModal}
               query={router.query.q || title}
+              fiche={`${router.route}/${router.query.slug}`}
```
Comment (revolunet) [inline packages/code-du-travail-frontend/src/common/Answer.js]: ce serait pas plutôt `url` que `fiche` ?

Comment (revolunet) [reply packages/code-du-travail-frontend/src/common/Answer.js]: `Router.asPath` (string) should work too

Comment (revolunet) [reply packages/code-du-travail-frontend/src/common/Answer.js]: wondering if `document.location.href`, at runtime, wouldnt be better so we also get hostname :)

Comment (lionelB) [reply packages/code-du-travail-frontend/src/common/Answer.js]: why not =)

**Review by douglasduteil — APPROVED** [review]

File: packages/code-du-travail-frontend/src/common/Answer.js (hunk 1 above)
Comment (lionelB) [reply packages/code-du-travail-frontend/src/common/Answer.js]: I've updated the component use `document.location.href` while submiting for using JS and component can also receive an url for server rendered case or when there is no javascript

**Review by revolunet — APPROVED** [review]

**Pull request merged**